src/Database/LSMTree/Internal.hs

```diff
@@ -1118,6 +1119,8 @@ data TableConfig = TableConfig {
     -- | Function for resolving 'Mupsert' values. This should be 'Nothing' for
     -- normal tables and 'Just' for monoidal tables.
   , confResolveMupsert   :: !(Maybe ResolveMupsert)
+    -- | The policy for caching key\/value data from disk in memory.
+  , confDiskCachePolicy  :: !DiskCachePolicy
```

Comment:
RE the commit message: 

> Add new 'confDiskCachePolicy' configuration option 

should be

> 

> Add new `confDiskCachePolicy` configuration option

Reply:
That's the same spelling. I'm confused. Or did you mean markup?

Reply:
Yup, markup